### PR TITLE
ECDC-2604: Add file writer and collector device to LoKI instrument

### PR DESCRIPTION
**Pull request by ebadkamil**
We added FileWriter and collector device to LoKi instrument. @kmurica also did a bug fix in FileWriter device that evaluates the absolute path for nexus config file.

### CODE REVIEW

File: nicos_ess/devices/datasinks/file_writer.py [hunk 1]
```diff
@@ -128,7 +133,7 @@ def doStart(self):
         session.log.info(f'Writing job with ID {self.job_id} is starting')
 
     def _read_nexus_config(self):
-        config = self.nexus_config_path
+        config = path.join(getNicosDir(), self.nexus_config_path)
```
Comment (mattclarke) [inline nicos_ess/devices/datasinks/file_writer.py:136]: This assumes that the file is somewhere in NICOS - that might not be true in production.

File: nicos_ess/requirements.txt [hunk 2]
```diff
@@ -12,3 +12,4 @@ PyQt5==5.15.4
 PyQtWebEngine==5.15.4
 QScintilla==2.12.1
 https://github.com/ess-dmsc/yuos_query/archive/v0.1.15.tar.gz
+https://github.com/ess-dmsc/file-writer-control/archive/v1.1.0.tar.gz
```
Comment (mattclarke) [inline nicos_ess/requirements.txt:15]: This is available from pip?

Comment (ebadkamil) [reply nicos_ess/requirements.txt]: yes, thanks. It is available from pip

**Comment by ebadkamil** [comment]
Would like to merge it if no further comments.

**Pull request merged**